src/components/views/sul-people/sul-people-table-view.tsx

```diff
@@ -1,15 +1,89 @@
+"use client";
+
+import { Table, Thead, Tbody, Tr, Th, Td } from 'react-super-responsive-table';
+import 'react-super-responsive-table/dist/SuperResponsiveTableStyle.css';
+import Link from 'next/link';
+import Image from "next/image";
+import {buildUrl} from "@/lib/drupal/utils";
+import LibCal from "@/components/node/stanford-person/libcal";
+import {EnvelopeIcon} from "@heroicons/react/20/solid";
+import EmailLink from "@/components/patterns/elements/email-link";
 import {NodeStanfordPerson} from "@/lib/gql/__generated__/drupal.d";
 
 interface Props {
   items: NodeStanfordPerson[]
   hasHeading: boolean
 }
 
-const SulPeopleTableView = async ({items, hasHeading}: Props) => {
+const SulPeopleTableView = async ({items, hasHeading }: Props) => {
+  const HeadingElement = hasHeading ? 'h3' : 'h2';
+  const imageUrl = items.suPersonPhoto?.mediaImage.url;
```

Comment:
@pookmish I'm not able to get the image to appear for the person on the grid. The error is `Property 'suPersonPhoto' does not exist on type 'NodeStanfordPerson[]'.` Do you see what is missing?

Reply:
`items` is an array what's why it's saying `NodeStanfordPerson[]`. It's an array of `NodeStanfordPerson` types.
You will have to map through the items and render each one separately.

There are some examples of how to do that you should be able to use.